### DISCUSSION TITLE
Accept DNS records under interactid subdomain

### DISCUSSION THREAD

**Discussion by brenocss** [projectdiscovery/interactsh#316] [Ideas]
### Please describe your feature request:

Accept DNS records under interactid subdomain
```
aws.{{interactid}}.{{interactserver}}
```

### Describe the use case of this feature:

An use case is to monitor if the DNS is being resolved and use it at the same time as the feature.

**Comment by MetzinAround** [comment]
@ehsandeep this is an old one, but I thought there might be a chance this was addressed in a release since this was posted. Just pinging you to check is this is possible with the latest version or not.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
